src/main/java/ch/epfl/biop/imaris/EasyXT.java

```diff
@@ -1241,6 +1243,66 @@ public static ISurfacesPrx create(ImagePlus imp, int timepoint) throws Error {
             return surface;
         }
 
+        /**
+         * create a new surfaces object from a Label ImagePlus
+         *
+         * @param impLabel the image to get a Surfaces from.
+         *                 A label image, each label will be a surface of the Surfaces object.
+         * @return the ISurfacesPrx with individual surface for each label value
+         * @throws Error an Imaris Error if there was a problem
+         */
+        public static ISurfacesPrx createFromLabels(ImagePlus impLabel) throws Error {
+            // Check if it has a Time Index Property
+            Object tInd = impLabel.getProperty("Time Index");
+            if (tInd != null) {
+                return createFromLabels(impLabel, (int) tInd);
+            }
+            log.warning("EasyXT cannot find a timepoint associated with this surface mask. Defaulting to Timepoint 0");
+            log.warning("Use Surfaces.create(ImagePlus imp, int timepoint) to specify the desired timepoint to instert this surface");
```

Comment:
instert typo

Reply:
corrected in 65ed3bdc72d68b632a02b29c958073032955a22f , which also corrects the older code